mobile/src/main/java/rs/readahead/washington/mobile/views/settings/SecuritySettings.kt

```diff
@@ -226,7 +226,7 @@ class SecuritySettings : BaseFragment() {
         if ((activity.getApplicationContext() as IUnlockRegistryHolder).unlockRegistry.getActiveMethod(
                 activity
             ) == UnlockRegistry.Method.TELLA_PIN
```

Comment:
This class is a mess :/, it needs a lot of cleaning and improvement.

Reply:
i agree, it needs improvement